src/main/java/dataloader/JsonDataLoader.java

```diff
@@ -0,0 +1,53 @@
+package dataloader;
+
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import tech.tablesaw.api.Table;
+import tech.tablesaw.io.json.JsonReadOptions;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+
```

Comment:
Documentation for class ??? :) :)

Reply:
it has separate documentations before each method. do you think a more general documentation is necessary?

Reply:
Well I would like to see the purpose of this class and that's why I asked for it...Although it is trivial perhaps for this class...

Reply:
I wrote some documentation on both `CsvDataLoader` & `JsonDataLoader` classes. check them out and tell me what you think.